docs/gateway-api/gateway-api-v1beta1.md

```diff
@@ -82,6 +82,12 @@ A sample Gateway object is shown below:
 
 The above Gateway object would correspond to a single Layer 7 virtual service in the AVI controller, with two ports (80, 443) exposed and a sslKeyAndCertificate created based on the secret **bar-example-com-cert**.
 
+The hostname field `.spec.listeners[i].hostname` is mandatory. It can configured with or without a wildcard, but cannot be only `*`.
```

Comment:
Might want to change it from "It can configured" to "It can be configured"

Reply:
fixed, thanks